controllers/ticketController.js

```diff
@@ -98,6 +107,11 @@ module.exports.disableTicket = async function (req, res, next) {
             dateCreated: ticket.dateCreated,
             updated: ticket.updated,
             //user: ticket.user,
+            //Iteration
+            // iterationUser: ticket.iteration.user,
+            iterationDateCreated: ticket.iteration.dateCreated,
+            iterationComment: ticket.iteration.comment,
```

Comment:
```suggestion
            //Iteration
            iteration: [
                ... ticket.iteration,
                {
                    //username: ticket.user,
                    dateCreated: new Date(),
                    comment: "Ticket disabled",
                    newStatus: ticketModel.TicketStatus.Cancelled
                }
            ]
```
Made it so when you go to disable a ticket, it will create a new iteration with the comment saying that they disabled the ticket.